data-model.sql

```diff
@@ -220,6 +210,28 @@ CREATE TABLE audit_patient (
 );
 ALTER TABLE audit_patient OWNER TO pulse;
 
+CREATE TABLE audit_document (
+	id bigserial not null,
+	audit_event_id bigint,
+	participant_object_type_code smallint NOT NULL, -- "2" (System)
```

Comment:
This may actually be a lookup table with a buch of values but we would only be using two. I can change that